pkg/cmd/command.go

```diff
@@ -57,6 +58,15 @@ func WithDir(dir string) Option {
 	}
 }
 
+// WithStdout returns an option that sets the cmd output to stdout.
+// This redirects output to stdout, should not use this option if you
+// need the return value of the command.
+func WithStdout() Option {
```

Comment:
The `runCommand` function will always return an error if this option is used. See https://github.com/golang/go/blob/master/src/os/exec/exec.go#L535-L537.